waftools/checks/custom.py

```diff
@@ -151,3 +151,13 @@ def fn(ctx, dependency_identifier, **kw):
             else:
                 return False
     return fn
+
+def check_cplugins_linkflags(ctx, dependency_identifier):
+    checks = [
+        check_cc(linkflags=['-rdynamic']),
+        check_cc(linkflags=['-Wl,--out-implib,libmpv.dll.a']) ]
```

Comment:
How exactly does this work? Does it require a libmpv.dll.a to be present?

Reply:
No, it's for generating an import lib from the exe to be used for linking cplugins.
If you are going to follow the
>mpv.exe a stub that just links against libmpv.dll
approach it won't be needed.

By the way, I noticed this breaks the build process when enabling `--enable-libmpv-shared`.
I'm sorry for the mess, I'm very unfamiliar with waf and should have thought better before pulling this request.